pkg/email/lore/parse.go

```diff
@@ -20,9 +22,28 @@ type Thread struct {
 	Messages  []*email.Email
 }
 
+type Series struct {
```

Comment:
Comment is needed. It seems to be the central object. Let's document it.
Alternatively you can add some "glossary" at the beginning of this file.

Reply:
Added a comment.